src/TestUtils/src/UITest.Appium/HelperExtensions.cs

```diff
@@ -1899,7 +1899,7 @@ static IQuery GetDefaultBackArrowQuery(IApp app)
 		{
 			return app switch
 			{
-				AppiumAndroidApp _ => AppiumQuery.ByXPath("//android.widget.ImageButton[@content-desc='Navigate up']"),
```

Comment:
Avoiding locale-dependent query issues. in a emulator/phone using Spanish language, this not works because the query would be something like: 
`//android.widget.ImageButton[@content-desc="Navegar hacia arriba"]`